sources/install/package_base.sh

```diff
@@ -173,6 +173,24 @@ function install_ultimate_vimrc() {
     sh ~/.vim_runtime/install_awesome_vimrc.sh
 }
 
+function install_neovim() {
+    if [[ $(uname -m) = 'x86_64' ]]
+    then
+        curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim.appimage
+        chmod u+x nvim.appimage
+        ./nvim.appimage --appimage-extract
+        cp -r squashfs-root/usr/ /opt/tools/nvim
+        rm -rf squashfs-root nvim.appimage
+    elif [[ $(uname -m) = 'aarch64' ]]
+    then
+        curl -LO https://github.com/neovim/neovim/releases/download/nightly/nvim-macos.tar.gz
```

Comment:
My bad. Neovim don't have ARM pre-build binary. We need to build.

Pseudo code here :
```bash
git clone https://github.com/neovim/neovim.git
cd neovim
make CMAKE_BUILD_TYPE=RelWithDebInfo
make install
```

Build take `~5min`

Reply:
> My bad. Neovim don't have ARM pre-build binary. We need to build.
> 
> Pseudo code here :
> 
> ```shell
> git clone https://github.com/neovim/neovim.git
> cd neovim
> make CMAKE_BUILD_TYPE=RelWithDebInfo
> make install
> ```
> 
> Build take `~5min`

https://github.com/ThePorgs/Exegol-images/pull/204/commits/825894fa0b5369600e1deb65e467ec73927bda57